coingaming/moon-design
# fix: console warnings [B1M2-3721_B1M2-3723_B1M2-3724]

### ---- TO TEST ---
- `DialogContent`(from: moon-design docs #B1M2-3721): Failed prop type: A <DialogContent> must have either an aria-label or aria-labelledby prop.

- `DialogContent`(from: moon-design docs #B1M2-3721):
React does not recognize the maxWidth prop on a DOM element

- `GeoMap`(from: moon-design docs #B1M2-3721)
Error: <path> attribute d: Expected number, "…008881569224855,NaNL23.008881569…".

**There is some problem with changing these components below**(they are all located in assets directory, and after `yarn build` they all go back to the previous version - that applies to [`LogoSportsbet`, `LogoBetaddaFull`, `PatternSpeed`, `CrestArsenal`, `CrestFlamengo`, `CrestSaoPaulo`, `CrestSouthampton`, `CrestWatford`]
- `LogoSportsbet`(from: portal promotions #B1M2-3724): React does not recognize the verticalAlign prop on a DOM element.
_**I have changed that also in these components, because the same error occured there:**_
`LogoBetaddaFull`

- `PatternSpeed`(from: portal promotions #B1M2-3724): React does not recognize the verticalAlign prop on a DOM element.

- `CrestArsenal`(from: portal promotions #B1M2-3724): 
Warning: Prop verticalAlign did not match. Server: "null" Client: "middle"
**_I have changed that also in these components, because the same error occured there:_**
`CrestFlamengo`
`CrestSaoPaulo`
`CrestSouthampton`
`CrestWatford`

### ---- DON'T OCCUR IN THE LIBRARY (so probably problem is in the apps) ----
- `Checkbox`(from: portal id #B1M2-3723): Warning: Prop id did not match. Server: "Checkbox-213iktp1o" Client: "Checkbox-pb3xocj5w
Problem when you use two checkboxes in same component

- `Button`(from: portal promotions #B1M2-3724): React does not recognize the iconLeft prop on a DOM element. 

### ---- NOT FIXED ----
- `Dialog`(from: portal id #B1M2-3723): React does not recognize the staticContext prop on a DOM element.
_Can't figure out what causes that and messing up the Dialog component isn't the way to fix it._
